.travis.yml

```diff
@@ -109,12 +109,31 @@ jobs:
           branch: master
           repo: denoland/deno
         skip-cleanup: true
+    
+    - name: "cargo release linux arm64"
+      os: linux
+      dist: xenial
+      script:
+      - rustup target add aarch64-unknown-linux-gnu
+      - sudo apt update
+      - sudo apt -yq --no-install-suggests --no-install-recommends install g++-5-aarch64-linux-gnu gcc-5-aarch64-linux-gnu g++-5-multilib libc6-arm64-cross
```

Comment:
Please wrap long lines at 80 columns

Unfortunately we don't have lint on travis.yml